inc/template-tags.php

```diff
@@ -50,10 +50,10 @@ function vantage_author_box( $post ) { ?>
  */
 function vantage_content_nav( $nav_id ) {
 	$jetpack_infinite_scroll_active = class_exists( 'Jetpack' ) && Jetpack::is_module_active( 'infinite-scroll' );
-	//Check if we're in the Page Builder Post Loop widget.
+	// Check if we're in the Page Builder Post Loop widget.
 	$is_page_builder_post_loop_widget = class_exists( 'SiteOrigin_Panels_Widgets_PostLoop' ) &&
-	                                    method_exists( 'SiteOrigin_Panels_Widgets_PostLoop', 'is_rendering_loop' ) &&
-	                                    SiteOrigin_Panels_Widgets_PostLoop::is_rendering_loop();
+										method_exists( 'SiteOrigin_Panels_Widgets_PostLoop', 'is_rendering_loop' ) &&
```

Comment:
Not too sure why this was changed. Maybe I'm missing something.

Reply:
My bad, the spacing fixes were changes that I hadn't pushed to develop when I created this branch. I guess this spacing should be spaces not tabs?
![template-tags_php_ _vantage](https://user-images.githubusercontent.com/789159/52293819-75c66600-2980-11e9-88e9-5f70ff3189b1.png)

Reply:
That's likely the issue here.. Ideally, use spaces for spacing as you can always rely on one space being one space while tabs and be displayed in many different ways (2 spaces, 4 spaces, etc).

Reply:
Will do.